src/style.css

```diff
@@ -198,3 +199,25 @@ footer,
   height: 20px;
   background-color: var(--bg-dark);
 }
+
+.modal-background {
+  height: 100%;
+  weight: 100%;
+  background-color: black;
+}
+
+#dialog {
+  width: 300px;
+  height: 300px;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  padding: 20px;
+}
```

Comment:
OPTIONAL: The modal window is too small. Let's try to make it a little bigger to make it at least 50% of the view width to accommodate the expansion of user comments. It must also be responsive to mobile view.